src/app/app.server.ts

```diff
@@ -0,0 +1,14 @@
+import express, { Express, Router } from "express";
+import { router as cakeRouter } from "./cake/cake.router";
+
+const app: Express = express();
+const router: Router = Router();
+
+app.use(router);
+router.use("/cakes", cakeRouter);
+
```

Comment:
You have missed the sell operations router

Reply:
Not yet deployed and already getting bugs jaja, thanks